.github/workflows/integration.yaml

```diff
@@ -38,4 +38,9 @@ jobs:
           # just sleep a bit to wait for the containers to start
           sleep 1
           docker-compose logs
-          go test ./... -tags=integration -count=1
+          go test ./... -tags=integration -count=1 -json | tee integration.json
+      - name: Run Gosec Security Scanner
+        run: |
+          curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $(go env GOPATH)/bin latest
+          gosec -fmt json -no-fail -out out.json ./...
+          head -n 30 out.json # to be removed
```

Comment:
Is this line supposed to be here now that the PR is final? 🙃

Reply:
My plan was to attach this PR to also upload this gosec json into local bubbly (running out of `docker-compose`). Apparently, thats going to take time  :) So removing the `todo`